package.json

```diff
@@ -22,6 +22,7 @@
     "core-js": "^3.6.5",
     "deep-object-diff": "^1.1.0",
     "dotenv": "^8.2.0",
+    "fibers": "^5.0.0",
```

Comment:
This is not a satisfying solution because fibers cannot be installed without a python environement. It was in optionalDependencies for the reason specified above:

![image](https://user-images.githubusercontent.com/6538906/101041563-4c9d9f00-3531-11eb-8f87-a5db382d9a1c.png)
From https://docs.npmjs.com/cli/v6/configuring-npm/package-json#optionaldependencies

I would try to bump the version to 5.x in the optional dependency and see if it works.